fileutil/fileutil.go

```diff
@@ -4,13 +4,37 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/bitrise-io/go-utils/pathutil"
 	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/pathutil"
 )
 
+// FileRemover ...
+type FileRemover interface {
```

Comment:
Do we have other interfaces in `go-utils` that operate on files? Should we rename this or merge it with existing interfaces? I expect other file operations in the future too.